mockfactory/desi/brick_pixel_quantities.py

```diff
@@ -80,34 +97,58 @@ def _one_brick(fn, ra, dec, dtype=None, default=None):
         return np.full(ra.size, default, dtype=dtype)
 
     # Load bricks class
+    from desiutil import brick
+    from astropy import wcs
     bricks = brick.Bricks()
 
-    # Create mpytools.Catalog to sort it across processes in the brick id
-    brickid_data = mpy.Catalog({'ra': ra, 'dec': dec, 'brickname': bricks.brickname(ra, dec), 'brickid': bricks.brickid(ra, dec)})
     # Create unique identification as index column
-    brickid_data['index'] = brickid_data.cindex()
-    # Copy unique identification to perform sanity check at the end
-    index = brickid_data['index']
+    cumsize = np.cumsum([0] + mpicomm.allgather(ra.size))[mpicomm.rank]
```

Comment:
this seems old stuff, no? (i.e. current version is the newest?)

Reply:
I restore the file .. ok I missed that.